spec/expression.html

```diff
@@ -119,8 +119,10 @@ <h1>Runtime Semantics: RecordPropertyDefinitionEvaluation</h1>
           1. Let _from_ be ! ToObject(_source_).
           1. Let _keys_ be ? _from_.[[OwnPropertyKeys]]().
           1. For each element _nextKey_ of _keys_ in List order, do
-            1. Let _value_ be _from_.[[Get]](_nextKey_).

```

Comment:
This was another bug: `[[Get]]` expects two parameters. I replaced it with a call to `Get()`, which seems to be used more often in the spec when the receiver matches the object.